content/blog/how-to-integrate-with-mainnet-protocols/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "How to Integrate with Ethereum Mainnet DeFi Protocols"
+date: "2020-04-24"
+description: "Wanna try build a dapp that interacts with multiple mainnet protocols but don't know where to start? Read on!"
+---
+
+While building out [Dedge](https://dedge.exchange), one of the first problems we encountered was:
+
+> How and where do we start integrating with mainnet DeFi protocols?
+
+Ideally, we would like to fork off the current mainnet state into a sandboxed environment, and develop/test our smart contracts from there. The emphasis on forking off the mainnet state it provides a couple key features:
```

Comment:
```suggestion
Ideally, we would like to fork off the current Mainnet state into a sandboxed environment, and develop/test our smart contracts from there. The reason this is important is because it provides a few essential benefits:
```